supervisory-authorities/derfhr.json

```
@@ -0,0 +1,16 @@
+{
+    "slug": "derfhr",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "Datenschutzbeauftragter Hessischer Rundfunk",
+    "address": "Bertramstraße 8\n60320 Frankfurt\nDeutschland",
+    "phone": "+49 69 1552541",
+    "email": "datenschutz@hr.de",
+    "web": "https://www.hr.de/",
```

Again taken from the Rundfunkdatenschutzkonferenz page:

```suggestion
    "web": "https://www.hr.de/unternehmen/struktur/datenschutz,ulrich_goehler-datenschutzbeauftragter-100.html",
```